PLEP-0008.rst

```diff
@@ -0,0 +1,163 @@
+with ``2023.4.1`` as the first bugfix release of the ``2023.4.*``
+series.
+
+Bugfix releases shall contain no changes to the API.
+
+Tradeoff Analysis
+-----------------
+
+Advantages of adopting calendar versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The age of a release can be gauged directly from the version number.
+
+* The extent of changes between releases of a package tends to be
+  roughly proportional to the time between releases, and can therefore
+  be gauged more accurately with date-based versioning.
+
+* The versioning scheme would be consistent with projects that use some
+  form of calendar versioning such as ``xarray``, ``dask``, ``pip``,
+  Ubuntu, and PyCharm.
+
+* The extent of changes between the last two releases in the ``0.*`` series
+  is likely to be comparable to the extent of changes between the last
+  ``0.*`` release and the ``1.0.0`` release. Hence, having a ``1.0.0``
+  release would indicate more changes than would actually be happening.
+  In calendar versioning, no ``1.0.0`` release will occur.
+
+* To computationally reproduce prior scientific results, it is sometimes
+  necessary to find a combination of historical releases of different
+  packages that are interoperable with each other. Because packages that
+  were released at about the same time are more likely to be
+  interoperable with each other, this process is simplified by using
+  calendar versioning.
+
+* Calendar versioning makes it easier to determine the appropriate time
+  to drop support for prior releases of packages, in particular if there
+  is a policy to support all releases made within a certain amount of
+  time in the past (e.g., `NumPy Enhancement Proposal 29`_).
+
+Disadvantages of adopting calendar versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Calendar versioning is not used by most of the packages PlasmaPy
+  depends on (i.e., NumPy, SciPy, matplotlib, and Astropy).
+
+* The version number itself would not indicate the maturity of the
+  package.
+
+* Calendar version numbers of PlasmaPy would differ substantially from
+  the version numbers of prior releases.
+
+* Changes to the version number would not indicate whether a release
+  contains backward incompatible changes, as in `semantic versioning`_.
+
+* The version number itself would not indicate whether a release is a
+  long-term support (LTS) release.
+
+* Versions of the form ``0.*.*`` (as currently in use) indicate that
+  breaking changes could occur for any release.
+
+Additional considerations for adopting calendar versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Calendar versioning is compatible with :pep:`440` and
+  ``setuptools_scm``.
+
+* Using the full year in the version number will reduce the chance of
+  ambiguity. Similarly, using the zero-padded month will increase the
+  likelihood that the minor version number is interpreted as a month.
```

Comment:
```suggestion
* Using the full year in the version number will reduce the chance of
  ambiguity. Similarly.
```

I'm just deleting this, but feel free to add back a comment about the month but without the "zero-padded" statement.